### PR TITLE
Added a bit of info about plugin root dir and shrinkwrap. For #243

### PR DESCRIPTION
**Pull request by MadsNielsen**
I think something is going on with the destination of the JPI files being written in a container...so print out those relevant informations during start.

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java [hunk 1]
```diff
@@ -104,6 +104,7 @@ public PluginManager configure(CNode config) throws ConfiguratorException {
         }
 
         File shrinkwrap = new File("./plugins.txt");
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java]: Storing it in a relative path looks dangerous. Why not just a fixed path in JENKINS_HOME?

File: src/test/resources/org/jenkinsci/plugins/casc/core/secret [hunk 2]
```diff
@@ -0,0 +1 @@
+s3cr3t
```
Comment (oleg-nenashev) [inline src/test/resources/org/jenkinsci/plugins/casc/core/secret]: unrelated? Or just a smoke test?

File: src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java (hunk 1 above)
Comment (MadsNielsen) [reply src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java]: I'd want to do that in a seperate pull request. For now im just interested to see what path is resolved inside my container.

File: src/test/resources/org/jenkinsci/plugins/casc/core/secret (hunk 2 above)
Comment (MadsNielsen) [reply src/test/resources/org/jenkinsci/plugins/casc/core/secret]: @oleg-nenashev oops :)

**Comment by ndeloof** [comment]
maybe use logger.debug ?

**Comment by MadsNielsen** [comment]
@ndeloof  I was originally going to do that, but it looks like java.util.Logger doesn't really have a `DEBUG` log level. So i just used the info as is used when presenting exception messages.

**Comment by ndeloof** [comment]
better to use `LOGGER.log(Level.CONFIG,  ...)` then

**Comment by ndeloof** [comment]
LGTM. Could you please squash and rebase your commits to keep a readable git history ?

**Comment by MadsNielsen** [comment]
Done @ndeloof

**Pull request merged**